packages/sidenav/package.json

```diff
@@ -18,6 +20,9 @@
     "react": "^16.8.6",
     "theme-ui": "^0.4.0-alpha.1"
   },
+  "devDependencies": {
+    "@types/mdx-js__react": "^1.5.1"
```

Comment:
Okay, but it's still a dev dependency. It doesn't exist for the end user.

Take a look here https://stackoverflow.com/a/46011417/6003547.

Reply:
Gotcha 😄 Updated that to be a dep 👍

Reply:
Done